dev-libs/smooth/smooth-0.8.74.0~pre4.recipe

```diff
@@ -0,0 +1,95 @@
+SUMMARY="A cross-platform C++ class library"
+DESCRIPTION="smooth is an object oriented C++ class library for Windows, \
+macOS, Linux and most Unix-like operating systems. It provides basic \
+functionality and platform support for applications and libraries.
+
+Features provided by smooth include:
+
+- user interface API with various widgets
+- simple to use multithreading API
+- file and network IO interface
+- completely transparent Unicode and software internationalization support
+- a libxml2 based XML parser"
+HOMEPAGE="http://www.smooth-project.org/"
+COPYRIGHT="1998-2018 Robert Kausch"
+LICENSE="Artistic v2.0"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/smooth/smooth-${portVersion/\~/-}.tar.gz"
+CHECKSUM_SHA256="815f0329b5546c78feeb9ad1813855dfcb7bc062e28de44d0b4aadd7c6625d80"
+SOURCE_DIR="smooth-${portVersion/\~/-}"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+fi
+
+libVersion="${portVersion%~*}"
+libVersionCompat="$libVersion compat >= ${libVersion%.*}"
+
+PROVIDES="
+	smooth$secondaryArchSuffix = $portVersion
+	lib:libsmooth_${libVersion%.*}$secondaryArchSuffix = $libVersionCompat
+	"
+if [ -z "$commandSuffix" ]; then
+PROVIDES+="
+	cmd:smooth_translator = $portVersion
+	"
+fi
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	smooth${secondaryArchSuffix}_devel = $portVersion
+	devel:libsmooth_${libVersion%.*}$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	smooth$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:make
+	cmd:gcc$secondaryArchSuffix
+	cmd:g++$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
```

Comment:
Could you sort these? A-Z

Reply:
The `cmd:` lines in `BUILD_PREREQUIRES` are now sorted.